.gitignore

```diff
@@ -24,3 +23,20 @@ selenium
 .babelcache.json
 *.swp
 *.swo
+
+# platform dependent parts of node_modules
```

Comment:
Stupid question, but why is this necessary? Will i have to have different seetings here bades on a windows, versus mac, vs any machine that has different global dependencies?